repackaging/images.yaml

```diff
@@ -103,3 +103,7 @@ images:
     type: python
     package: redis-mcp-server
     version: 0.3.0
+  - name: ref
+    type: node
+    package: ref-tools-mcp
+    version: 3.0.1
```

Comment:
The latest version shows 3.0.0 https://www.npmjs.com/package/@iflow-mcp/ref-tools-mcp maybe I am missing something here?

Reply:
oh I guess this is the correct npmjs page: https://www.npmjs.com/package/ref-tools-mcp, not the iflow-mcp one